release/src-rt-6.x.4708/linux/linux-2.6.36/config_base.6a

```diff
@@ -557,7 +557,7 @@ CONFIG_NETFILTER_XT_MATCH_MAC=y
 CONFIG_NETFILTER_XT_MATCH_MARK=y
 CONFIG_NETFILTER_XT_MATCH_MULTIPORT=y
 # CONFIG_NETFILTER_XT_MATCH_OSF is not set
-# CONFIG_NETFILTER_XT_MATCH_OWNER is not set
```

Comment:
Is not set

Reply:
Yes, it was not set and required for specific iptables rules